packages/solana/src/connectors/walletAdapters.ts

```diff
@@ -0,0 +1,39 @@
+import { PhantomWalletAdapter } from '@solana/wallet-adapter-phantom'
```

Comment:
In this folder (`/connectors`), this has `camelCase` while some others have `KebabCase`, let's stick to same naming convention (in case there is something different with those, but as I see both contain a class)

Reply:
@valentyn-solana Sorry I meant about the file names 🙏